code/modules/atmospherics/pipes.dm

```diff
@@ -194,6 +196,26 @@
 	else
 		. = PROCESS_KILL
 
+/obj/machinery/atmospherics/pipe/simple
+	icon = 'icons/atmos/pipes.dmi'
+	icon_state = "11"
+	var/pipe_icon = "" //what kind of pipe it is and from which dmi is the icon manager getting its icons, "" for simple pipes, "hepipe" for HE pipes, "hejunction" for HE junctions
```

Comment:
Comment seems somewhat out of date from the discussions about killing the icon cache/icon manager.

Reply:
I believe this var can be fully removed, will look into it.